src/runner/index.js

```diff
@@ -68,10 +68,10 @@ export default class Runner {
         return this;
     }
 
-    reporter (reporter, outStream = null, formatter = null) {
+    reporter (reporter, outStream = null, errorDecorator = null) {
         this.bootstrapper.reporter = reporter;
         this.opts.reportOutStream  = outStream;
-        this.opts.formatter        = formatter;
+        this.opts.errorDecorator   = errorDecorator;
```

Comment:
We need to pass this decorator to the reporter's ctor in the bootstrapper as well